subjects/ascii-fs.en.md

```diff
@@ -0,0 +1,47 @@
+## ascii-art-fs
+
+### Objectives
+
+Ascii-art-fs consists on receiving two `strings` has arguments. The first string being a random phrase and the second string being the template for the graphic representation in ascii of the first argument.
```

Comment:
We usually inline code like Go types : `int`, `bool`, `string` but not `strings` since it's not a Go type, so I would either let "strings" or write `string`